packages/ckeditor5-image/src/image/utils.js

```diff
@@ -133,6 +147,58 @@ export function getViewImgFromWidget( figureView ) {
 	return figureChildren.find( viewChild => viewChild.is( 'element', 'img' ) );
 }
 
+// Creates a view element representing the image.
+//
+//		<figure class="image"><img></img></figure>
+//
+// Note that `alt` and `src` attributes are converted separately, so they are not included.
+//
+// @private
+// @param {module:engine/view/downcastwriter~DowncastWriter} writer
+// @param {'image'|'imageInline'} imageType The type of created image.
+// @returns {module:engine/view/containerelement~ContainerElement}
+export function createImageViewElement( writer, imageType ) {
```

Comment:
This is an exported function so it should have a proper JsDoc. And should not be private. And it should describe the imageInline HTML.

Reply:
Done.